helix-core/src/test/java/org/apache/helix/tools/TestHelixAdminCli.java

```diff
@@ -600,6 +602,15 @@ public void testDeactivateCluster() throws Exception {
       }
     }
 
+    boolean leaderNotExists = TestHelper.verify(() -> {
+      boolean isLeaderExists = _gZkClient.exists(path);
+      if (isLeaderExists) {
+        System.out.println("mysteriou leader out");
```

Comment:
typo? "mysteriou"

This print is too verbose. Please remove.

Reply:
removed.